src/css/components/app-list.less

```diff
@@ -42,10 +42,35 @@
   }
 
   & tr td:first-child,
-  & tr th:first-child
-  {
+  & tr th:first-child {
     padding-left: @base-spacing-unit*2;
   }
+
+  & > thead > tr > th {
```

Comment:
Yeah I hear you. Thing is, we want to make sure we are in the application table's header columns - and I was sticking to the existing "nesting style". Mhmh.